setup.py

```diff
@@ -75,29 +75,6 @@ def run(self, *args, **kwargs):
         self.run_command("buildhtml")
 
 
-def _config_exe(exe_name):
-    """Sets the environment to run the real executable (returned)"""
-
-    package_name = "nmodl"
-
-    assert (
-        package_name in working_set.by_key
-    ), "NMODL package not found! Verify PYTHONPATH"
-    NMODL_PREFIX = os.path.join(working_set.by_key[package_name].location, "nmodl")
-    NMODL_PREFIX_DATA = os.path.join(NMODL_PREFIX, ".data")
-    if sys.platform == "darwin":
-        os.environ["NMODL_WRAPLIB"] = os.path.join(
-            NMODL_PREFIX_DATA, "libpywrapper.dylib"
-        )
-    else:
-        os.environ["NMODL_WRAPLIB"] = os.path.join(NMODL_PREFIX_DATA, "libpywrapper.so")
-
-    # find libpython*.so in the system
-    os.environ["NMODL_PYLIB"] = find_libpython()
-
-    return os.path.join(NMODL_PREFIX_DATA, exe_name)
```

Comment:
@alkino : are you about this? i.e. this is tested? (travis part is obvious but wondering about this!)

Reply:
oh, this function is not used (or this is really black magic).

Reply:
We can found a copy there: https://github.com/BlueBrain/nmodl/pull/1036 (and this one is used).

Reply:
Ok, I thought this must be used under binary shim but I see a copy here: https://github.com/BlueBrain/nmodl/blob/07c7267998e2066f744885e43b88a595aa5f5591/pywheel/shim/_binwrapper.py#L14

Reply:
Sorry, my link was broken